src/actions/index.js

```diff
@@ -28,10 +27,11 @@ export function getResponseJSON(response) {
   return response.json();
 }
 
-export function requestErrorHandler(dispatch, fetchAction) {
+export function requestErrorHandler() {
   return (err) => {
-    const callName = fetchAction ? fetchAction.type : "";
-    notifyError(getMessage("APICallFailed").replace('{callName}', callName).replace('{err}', err));
+    // eslint-disable-next-line
```

Comment:
Why are we disabling ESlint altogether here? If it's because Raven is not defined, [specify it as a global](https://eslint.org/docs/user-guide/configuring#specifying-globals) or better yet, import it.
```javascript
import Raven from 'raven-js';
```

Reply:
Fixed

Reply:
Is the eslint directive redundant now? Or necessary? I got confused 😕

Reply:
Yes it's redundant and was removed as Raven is imported and thus defined now.

Reply:
But isn't it still active for that following line (numbered 32 and 33 I think) on this change?
Or am I looking at this wrong..?